data/mods/Xedra_Evolved/items/ammo.json

```
@@ -0,0 +1,15 @@
+[
+  {
+    "id": "ss_57mm",
+    "copy-from": "57mm",
+    "type": "AMMO",
+    "name": { "str_sp": "5.7x28mm, subsonic" },
```

```suggestion
    "name": { "str_sp": "5.7x28mm SB193" },
```
https://www.fnherstal.com/en/defence/ammunition/57x28mm/
Shouldn't this be vanilla content too?

---

I'm happy to add it into vanilla.   I was hoping you'd give me better info and let me know if my changes on the real ammo and imagined pistol were accurate/reasonable.  Thank you for giving me the right name :)